preprocess/code/tests/test_update_preprocess.py

```diff
@@ -0,0 +1,300 @@
+        var_err = VariableDisplayUtil(pre_file, self.update_json_01).error_messages
+        self.assertEqual(ValueError,var_display_modify)
+        print("Error : ",var_err)
+
+    def test_50_update(self):
+        """(50) test for variable display section not found in preprocess file"""
+        msgt(self.test_50_update.__doc__)
+        pre_file= {
+               "$schema":"http://(link to eventual schema)/jjonschema/1-0-0#",
+               "self":{
+                  "description":"TwoRavens metadata generated by ....",
+                  "created":"..time stamp..",
+                  "preprocess_id":45,
+                  "data_url":"http://metadata.2ravens-url.org/preprocess/data/45",
+                  "format":"jsonschema",
+                  "preprocess_version":"1-0-0",
+                  "schema_version":"1.0.0"
+               },
+               "variables": {
+                  "cylinders": {
+                     "numchar":"continuous",
+                      "nature": "nominal",
+                     "mean":213,
+                     "median":34
+
+                  },
+                  "mpg":{
+                      "numchar": "continuous",
+                       "nature": " ordinal",
+                       "mean": 313,
+                       "median": 54
+
+                   }
+               }
+}
+        var_display_modify = VariableDisplayUtil(pre_file, self.update_json_01).var_display()
+        var_err = VariableDisplayUtil(pre_file, self.update_json_01).error_messages
+        self.assertEqual(ValueError, var_display_modify)
+        print("Error : ",var_err)
+
+    def test_60_update(self):
+        """(60) test for variable display section not found in preprocess file"""
+        msgt(self.test_60_update.__doc__)
+        update = {
+            "preprocess_id": 45
+        }
+        var_display_modify = VariableDisplayUtil(self.preprocess_json_01, update).var_display()
+        var_err = VariableDisplayUtil(self.preprocess_json_01,update).error_messages
+        self.assertEqual(ValueError, var_display_modify)
+        print("Error : ",var_err)
+
+    def test_70_update(self):
+        """(70) test for variable display section's viewable not found in update file"""
+        msgt(self.test_70_update.__doc__)
+        update = {
+            "preprocess_id": 45,
+            "variable_updates": {
+               "cylinders" : {
+                 "omit": ["mean", "median"],
+                 "label": {
+                     "numchar":"discrete",
+                     "nature": "ordinal"
+                 }
+               },
+               "mpg": {
+                 "viewable": "false",
+                 "omit": [],
+                 "label": {
+
+                 }
+               }
+            }
+        }
+
+        var_display_modify = VariableDisplayUtil(self.preprocess_json_01, update).var_display()
+        var_err = VariableDisplayUtil(self.preprocess_json_01, update).error_messages
```

Comment:
this is starting the whole process over again.  take the `var_display_modify` object from above and check for errors.

```python
assertTrue(var_display_modify.has_error)
```